modules/nf-core/cellranger/vdj/main.nf

```diff
@@ -0,0 +1,54 @@
+process CELLRANGER_VDJ {
+    tag "$meta.gem"
+    label 'process_high'
+
+    container "nfcore/cellranger:7.1.0"
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "CELLRANGER_VDJ module does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+    input:
+    tuple val(meta), path(reads)
+    path  reference
+
+    output:
+    tuple val(meta), path("sample-${meta.gem}/outs/*"), emit: outs
+    path "versions.yml"                               , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def sample_arg = meta.samples.unique().join(",")
+    def reference_name = reference.name
+    """
+    cellranger \\
+        vdj \\
+        --id='sample-${meta.gem}' \\
```

Comment:
Why are you prefixing with "-sample"?

Reply:
also the style from `cellranger count`:

https://github.com/nf-core/modules/blob/ffaff8089fa2dec69e52a18c72384eee32bea2c4/modules/nf-core/cellranger/count/main.nf#L30

also happy to change, but then we should change `cellranger count` too

Reply:
went ahead and removed `sample-` prefix 👍🏼